ge.py

```diff
@@ -0,0 +1,62 @@
+# this code here generate command for the autocompletion.
+
+"""
```

Comment:
you can remove useless commented codes I think

Reply:
that just explains it so I can remember what the file does